code/modules/research/rdmachines.dm

```diff
@@ -48,7 +48,7 @@
 	desc = "A machine used for printing advanced circuit boards. Operated from an R\&D console."
 	icon_state = "imprinter"
 	circuit = /obj/item/circuitboard/circuit_imprinter
-
+	mat_efficiency = 0.2
```

Comment:
```suggestion
	mat_efficiency = 0.1
```

Reply:
Don't think this actually will change anything since it only has 1 manip